source/renderer/app/components/loading/system-time-error/SystemTimeError.tsx

```diff
@@ -104,17 +104,20 @@ class SystemTimeError extends Component<Props> {
     const isNTPServiceReachable = !!localTimeDifference;
     const allowedTimeDifferenceInSeconds = ALLOWED_TIME_DIFFERENCE / 1000000;
     const rawTimeOffset = (localTimeDifference || 0) / 1000;
-    const timeOffset = humanizeDurationByLocale(rawTimeOffset, currentLocale, {
-      delimiter: ' ',
-      units: ['y', 'mo', 'w', 'd', 'h', 'm', 's', 'ms'],
-      localeConfig: {
-        'ja-JP': {
-          spacer: '',
-          delimiter: '',
-          serialComma: false,
-        },
-      },
-    });
+    const timeOffset = currentLocale
```

Comment:
In what cases `currentLocale` may not be set? Would it be better to ensure that currentLocale is always provided (or alternatively provide a fallback value) rather than using `null`?

Reply:
Well, it may be set in all cases, but it may become undefined for unexpected reasons, and therefore crash the component. Such as in this case, in which it probably worked correctly when it was set up but some update to some library might have changed the requirements for it to work, and then it crashed. So I would set defensive code as in here, on the cases in which it makes sense.